Update New Membership Coordinators in OWNERS files

nikhita opened the pull request:
ref: https://github.com/kubernetes/community/pull/4995

/hold
until https://github.com/kubernetes/community/pull/4995 merges

Since the whole roster of NMCs is new, this adds the new NMCs as reviewers in OWNERS files. They will be moved to approvers after a month. :+1: 

I haven't added new folks to k-incubator because we intend to shut down the org.

/assign @cblecker @mrbobbytables 
cc @ameukam @palnabarun @savitharaghunathan

cblecker commented:
/lgtm

mrbobbytables commented:
/pony party
/honk
/approve

mrbobbytables commented on config/kubernetes-sigs/org.yaml:
```diff
@@ -404,6 +404,7 @@ members:
 - sanchezl
 - SandeepPissay
 - saschagrunert
+- savitharaghunathan
 - SataQiu
```
These two should be flipped^^;;;;

idealhack commented:
/approve

nikhita commented:
> /hold I had discussed w/ @cblecker moving out of the review path, but retaining approval for other GH Admin tasks (like team cleanup).

@justaugustus I've removed all old NMCs as part of this PR and added new ones. Let's discuss in another issue/PR about updating approvers? The current policy mentions only GitHub Admin team + NMCs for approvers to keep it tightly confined.

mrbobbytables commented:
/lgtm

nikhita commented:
/hold cancel
